fingerprint/capture/src/main/java/com/simprints/fingerprint/capture/screen/FingerprintCaptureFragment.kt

```diff
@@ -104,9 +111,26 @@ internal class FingerprintCaptureFragment : Fragment(R.layout.fragment_fingerpri
             args.params.fingerprintSDK,
         )
 
+        scanningStatusTracker.scanCompleted.observe(viewLifecycleOwner, LiveDataEventObserver {
+            if (isAudioEnabled())
+                playBeep()
+        })
         initUI()
     }
 
+    private fun playBeep() {
+        val mediaPlayer = MediaPlayer.create(context, R.raw.beep)
+        mediaPlayer.start()
+        mediaPlayer.setOnCompletionListener {
+            it.release()
+        }
+    }
+
+    private fun isAudioEnabled() =
+        PreferenceManager
+            .getDefaultSharedPreferences(requireContext())
+            .getBoolean(AUDIO_PREFERENCE_KEY, true)
```

Comment:
I haven't yet reached an agreement with Jon regarding the default value of the preference.

Reply:
The final agreement is to make it false